### DISCUSSION TITLE
Can't compile, because of missing files.

### DISCUSSION THREAD

**Discussion by HansNelisse-SH** [hathach/tinyusb#1773] [Q&A]
### Operating System

Windows 10

### Board

Raspberry Pico

### Firmware

Not sure

### What happened ?

I took the steps mentioned here (https://docs.tinyusb.org/en/latest/reference/getting_started.html)

I've cloned tinyusb to a local folder. Included tusb.c into my project. 
When I compile I get an error in osal_pico.h about unable to find "pico/time.h".
Indeed this file can't be found on my computer.

### How to reproduce ?

See above

### Screenshots

![image](https://user-images.githubusercontent.com/112565726/205307286-f037623c-7fc4-478a-8e3a-74fbd5a6467b.png)

### I have checked existing issues, dicussion and documentation

- [X] I confirm I have checked existing issues, dicussion and documentation.

**Comment by hathach** [comment]
You should try to follow rpi official docs and their getting started first to make sure you can compile and test with make/cmake/gcc. Using IDE is out of support scope here. If you really want IDE, try to read and mimic make/cmake contents to add src files, path and defines, cflags accordingly.

**Reply by HansNelisse-SH** [reply]
I wasn't aware that I needed the Pico SDK. That's where I found the missing files. 
A while ago I played with the SDK and it took me days of strugling to get a simple LED blink to work. I'm not familiar yet with all this make/cmake stuff. Whereas with the Segger IDE it only took minutes, and I could also realtime debug it. 
I'll give it another try though.
